io.github.Sunderland93.sway-input-config.yml

```diff
@@ -0,0 +1,71 @@
+id: io.github.Sunderland93.sway-input-config
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm15
+command: sway-input-config
+rename-desktop-file: sway-input-config.desktop
+rename-icon: sway-input-config
+finish-args:
+- "--socket=wayland"
+- "--device=dri"
+- "--filesystem=/run/user:ro"
```

Comment:
> Ok, I assume for this:
> 
> ```
> Traceback (most recent call last):
>   File "/app/bin/sway-input-config", line 5, in <module>
>     from sway_input_config.main import main
>   File "/app/lib/python3.10/site-packages/sway_input_config/main.py", line 23, in <module>
>     sway_version = get_sway_version()
>   File "/app/lib/python3.10/site-packages/sway_input_config/utils.py", line 6, in get_sway_version
>     i3 = Connection()
>   File "/app/lib/python3.10/site-packages/i3ipc/connection.py", line 68, in __init__
>     raise Exception('Failed to retrieve the i3 or sway IPC socket path')
> Exception: Failed to retrieve the i3 or sway IPC socket path
> ```
> 
> Would be nice to have a proper UI error for when there's no socket found instead of an exception in the terminal.

Ok, I'll try to add it in the next release

Reply:
Should be fixed now